### PR TITLE
Parameterize the wallet file location

### PR DESCRIPTION
**Pull request by andozw**
Allows for setting custom wallet file locations and running multiple instances in the same environment.

### CODE REVIEW

File: java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java
Comment (ivmaykov) [inline java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java:59]: nit: I think this should be called `--wallet-dir` or similar since it's actually the path to the wallet directory

Comment (ivmaykov) [reply java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java:59]: nit: and similarly call the variable `walletDirectory` or something like that, since it's neither a Path object (it's a String), nor names the wallet file.

File: java/gui/src/main/java/com/squareup/subzero/wallet/WalletLoader.java
Comment (ivmaykov) [inline java/gui/src/main/java/com/squareup/subzero/wallet/WalletLoader.java:35]: super nit: this can call the other constructor instead: `this(FileSystems ...)`, this way if the Path constructor gets more logic later we don't need to duplicate it here.

File: java/gui/src/test/java/com/squareup/subzero/wallet/WalletLoaderTest.java
Comment (ivmaykov) [inline java/gui/src/test/java/com/squareup/subzero/wallet/WalletLoaderTest.java:114]: super nit: should we also add the following test cases?
- path looks well-formed but doesn't exist
- path refers to a regular file rather than a directory

**Review by ivmaykov — DISMISSED** [review]
Approving with some nits, please fix them before merging.

File: java/gui/src/test/java/com/squareup/subzero/wallet/WalletLoaderTest.java
Comment (andozw) [reply java/gui/src/test/java/com/squareup/subzero/wallet/WalletLoaderTest.java:114]: i didn't include these because they get handled by the marker file check, but it doesn't hurt to have the tests explicitly in case something changes later.

**Review by ivmaykov — APPROVED** [review]
LGTM

**Pull request merged**
